include/xtensor/xmath.hpp

```diff
@@ -2118,11 +2118,10 @@ namespace xt
         XTL_REQUIRES(xtl::negation<is_reducer_options<X>>, xtl::is_integral<D>)>
     inline auto variance(E&& e, X&& axes, const D& ddof, EVS es = EVS())
     {
-        decltype(auto) sc = detail::shared_forward<E>(e);
         // note: forcing copy of first axes argument -- is there a better solution?
         auto axes_copy = axes;
         // always eval to prevent repeated evaluations in the next calls
-        auto inner_mean = eval(mean<T>(sc, std::move(axes_copy), evaluation_strategy::immediate));
```

Comment:
That is the original implementation of `variance` over axis.
Here the shared forward is removed as it is unnecessary (due to `eval`).